docs/content/documentation/templates/overview.md

```diff
@@ -154,6 +160,12 @@ If you only need the metadata of the section, you can pass `metadata_only=true`
 {% set section = get_section(path="blog/_index.md", metadata_only=true) %}
 ```
 
+If selecting a specific language for the section, you can pass `lang` with the language code to the function:
```

Comment:
I would mention for pages and here that it's equivalent to `{% set section = get_section(path="blog/_index.fr.md") %}`